includes/abstracts/abstract-wp-job-manager-form.php

```diff
@@ -322,7 +322,6 @@ public function clear_fields() {
 	 * Enqueue the scripts for the form.
 	 */
 	public function enqueue_scripts() {
```

Comment:
Is this actually called from somewhere?

Reply:
Oh looks like I had the warning about this because it's called in the Applications add-on's form (`WP_Job_Manager_Applications_Apply`) that extends this class. Since this is a generic base class used all around I'd keep the method even if we don't use it now in core.